client/src/core/client/admin/components/ModerationReason/Reasons.tsx

```diff
@@ -0,0 +1,102 @@
+import { Localized } from "@fluent/react/compat";
+import React, { FunctionComponent } from "react";
+
+import { GQLREJECTION_REASON_CODE } from "coral-framework/schema";
+import { /* Button, */ RadioButton } from "coral-ui/components/v2";
+import { Button } from "coral-ui/components/v3";
+
+import { unsnake } from "./formatting";
+
+import styles from "./ModerationReason.css";
+
+export interface Props {
+  onCode: (code: GQLREJECTION_REASON_CODE) => void;
+  selected: GQLREJECTION_REASON_CODE | null;
+  onAddExplanation: () => void;
+}
+
+const AddExpanationButton: FunctionComponent<{ onClick: () => void }> = ({
+  onClick,
+}) => (
+  <Localized id="common-moderationReason-addExplanation">
+    <Button
+      onClick={onClick}
+      className={styles.optionAction}
+      variant="none"
+      color="success"
+    >
+      + Add explanation
+    </Button>
+  </Localized>
+);
+
+const Reasons: FunctionComponent<Props> = ({
+  selected,
+  onCode,
+  onAddExplanation,
+}) => {
+  return (
+    <>
+      <Localized id="common-moderationReason-reason">
+        <span className={styles.sectionLabel}>Reason</span>
+      </Localized>
+
+      {Object.values(GQLREJECTION_REASON_CODE)
+        .filter(
+          (code) =>
+            code !== GQLREJECTION_REASON_CODE.ILLEGAL_CONTENT &&
+            code !== GQLREJECTION_REASON_CODE.OTHER
+        )
+        .map((code) => (
+          <>
```

Comment:
I think the unique `key` prop needs to be moved up here to address the error in the console for this.